subprojects/docs/src/main/resources/header.html

```diff
@@ -74,6 +74,11 @@
             <li><a href="/userguide/userguide.html">User Manual Home</a></li>
             <li><a href="/release-notes.html">Release Notes</a></li>
             <li><a href="/userguide/installation.html">Installing Gradle</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#upgrading-gradle" aria-expanded="false" aria-controls="upgrading-gradle">Upgrading Gradle</a>
+                <ul id="upgrading-gradle">
+                    <li><a href="/userguide/upgrade_version_4.html">version 4.x to 5.0</a></li>
```

Comment:
I'm not sure "Upgrading builds" conveys the intent more clearly. "Upgrading how?", I ask. I fear users may miss it thinking it's something that it's not.

I would feel differently if the nav were expanded because "Upgrading builds from Gradle 4.x to 5.0" as one statement has a nice ring to it. 

Do you agree?

Reply:
For me, it still comes down to the fact that upgrading Gradle is the process of installing a new version of it. But, I imagine most folks will view it as intended, i.e. how to upgrade your build to new versions of Gradle.

I would leave it as is, although the lower case "version" looks odd to me when all the other nav entries are capitalised.